src/libraries/System.Collections.Concurrent/src/System.Collections.Concurrent.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <OmitResources>false</OmitResources>
```

Comment:
OmitResources set to false means resources are emitted?  They weren't already being emitted?

Reply:
I see, you changed the condition such that by default partial facades don't get resources.  That seems a little wonky, that a library could define a Strings.resx but by default it wouldn't be used.

Reply:
It would fail the build if they tried to use it.

Reply:
But why isn't the condition by default just to emit resources if there are any and not if there aren't?

Reply:
Often libraries are built as a partial facade in some configurations but not others (ie., there is a condition on IsPartialFacadeAssembly). We only want to skip the resx automatically when they are built as a partial facade. That would not occur if we triggered based on the existence of the resx, which is in a location shared by all configurations.

Reply:
Ok.  FWIW, I find the double-negative super confusing, especially since emit and omit sound so much alike.

Reply:
Yes, I find negative sense booleans super confusing. Maybe this could be reversed in sense in future.

Reply:
I got very confused by this as well 😄